### ISSUE TITLE
Wrong signature for addCallbacks in the Deferred Reference

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2247]
|[<img alt="zseil's avatar" src="https://avatars.githubusercontent.com/u/421894?s=50" width="50" height="50">](https://github.com/zseil)| @zseil reported|
|-|-|
|Trac ID|trac#2247|
|Type|defect|
|Created|2006-11-26 23:55:52Z|

The signature for addCallbacks in the [Using Deferreds howto](http://twistedmatrix.com/projects/core/documentation/howto/defer.html#auto10) contains a nonexistent asDefaults argument and is missing a
callbackKeywords argument. Patch attached.

Attachments:

* [twisteddeferdoc.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/daa/daa446c9925db212579eae2db081b4bc6bdf1091/3b483c339dbbd813a82f712beffc718e592f203c.diff) (551 bytes) - added by zseil on 2006-11-26 23:58:18Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2247 2247
type__defect defect
reporter__zseil zseil
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1164585352000000 1164585352000000
changetime__1173360625000000 1173360625000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve set status to `closed`|
|-|-|

(In [19781]) Correct signature of addCallbacks in Deferred reference documentation

Author: zseil
Reviewer: therve
Fixes [#2247](https://github.com/twisted/twisted/issues/2247)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
